pkgs/by-name/go/goldwarden/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
+, libfido2
+, dbus
+, pinentry
+}:
+
+buildGoModule rec {
+  pname = "goldwarden";
+  version = "0.2.7";
+
+  src = fetchFromGitHub {
+    owner = "quexten";
+    repo = "goldwarden";
+    rev = "v${version}";
+    hash = "sha256-OXJovoJ2+YIMqzoG6J2LlxUC5DMZRAdEl+ZEv6PDXlI=";
+  };
+
+  vendorHash = "sha256-1Px60+f23qoP5eEOUC3WG5vKJYjbD3bPOrDyBpXlMT0=";
+
+  ldflags = [ "-s" "-w" ];
+
+  nativeBuildInputs = [makeWrapper];
```

```suggestion
  nativeBuildInputs = [makeBinaryWrapper];
```
preferred when possible, per nixpkgs manual

---

Done.